builders/integrations/wallets/rainbow-kit.md

```diff
@@ -0,0 +1,209 @@
+
+### Set Custom Initial Chain
+
+RainbowKit will connect by default to the first chain supplied to Wagmi in the config. If you compare the order of chains listed in `wagmi.ts` to those on the **Switch Networks** modal, you will see they are the same. A simple fix would be to move `moonbaseAlpha` to the top of the chain list; however, relying on this default behavior is not the most reliable option. 
+
+Instead, you can use the `initialChain` prop that is part of the `RainbowKitProvider` element to define which chain the wallet should initially connect to when the user selects **Connect Wallet**. Open your `providers.tsx` file and update the code to configure the `initialChain` prop. You can pass either a chain ID or chain name from the [Wagmi Chains list](https://wagmi.sh/core/api/chains){target=\_blank}.
+
+```js title="providers.tsx"
+--8<-- 'code/builders/integrations/wallets/rainbowkit/initial-chain.ts'
+```
+### Define Custom Theme Colors
+
+RainbowKit offers three built-in theme functions: `lightTheme`, `darkTheme`, and `midnightTheme`. These theme functions return a theme object, which you can pass into the `RainbowKitProvider` prop `theme` to set custom colors, border radius, font stack, and overlay blur. Update `providers.tsx` with the following code. Be sure to add `darkTheme` to the `@rainbow-me/rainbowkit`import statement to allow your changes to render correctly.
+
+```js title="providers.tsx"
+--8<-- 'code/builders/integrations/wallets/rainbowkit/custom-theme.ts'
+```
+
+## Handle Disconnections
+
+You can now disconnect MetaMask from your dApp and then reconnect to test your customizations. There are two options for completing this step.  
+
+### Disconnect from DApp {: #disconnect-from-dapp }
+
+RainbowKit includes a **Disconnect** button out of the box. To open the modal, select the arrow next to your account number. Click the **Disconnect** button. You should now see **Connect Wallet**; your account information should no longer be visible. 
+
+![Built in Disconnect button](/images/builders/integrations/wallets/rainbowkit/rainbowkit-2.webp)
+
+### Disconnect from MetaMask Mobile {: #disconnect-from-metamask-mobile }
+
+Some users prefer to disconnect from their mobile wallet rather than use a button within a dApp. To use this method: 
+
+- click on the MetaMask extension in your browser to open the modal
+- click the three dots in the upper right corner of the MetaMask modal
+- select **Connected sites**
+- review the list of sites connected to your wallet
+- select **Disconnect** for each site you want to disconnect
+
```

Comment:
These would be numbered steps, and the first letter of each step should be capitalized

Reply:
updated